examples/api-samples/src/browser/view/sample-unclosable-view-contribution.ts

```diff
@@ -14,15 +14,29 @@
  * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
  ********************************************************************************/
 
-import { AbstractViewContribution, bindViewContribution, WidgetFactory } from '@theia/core/lib/browser';
+import { inject, injectable, interfaces } from 'inversify';
+import { AbstractViewContribution, bindViewContribution } from '@theia/core/lib/browser/shell/view-contribution';
+import { TabBarToolbarContribution, TabBarToolbarRegistry } from '@theia/core/lib/browser/shell/tab-bar-toolbar';
+import { Command, CommandRegistry, Emitter, MessageService } from '@theia/core/lib/common';
+import { Widget, WidgetFactory } from '@theia/core/lib/browser';
 import { SampleViewUnclosableView } from './sample-unclosable-view';
-import { injectable, interfaces } from 'inversify';
+
+export const SampleToolBarCommand: Command = {
+    id: 'navigator.toggle.sampleToolbarCommand',
+    category: 'Sample',
```

Comment:
`category` can be removed since it's not longer present in the quick-command palette.